File: AOJ/0407/meeting_place.c
#include <stdio.h>

int main() {
  int i, N, x, xmin = 2000, xmax = 0;
  scanf("%d", &N);

  for ( i = 0; i < N; i++ ) {
    scanf("%d", &x);
    if ( x < xmin ) xmin = x;
    if ( x > xmax ) xmax = x;
  }

  printf("%d\n", xmax - (xmin + xmax) / 2);
  return 0;
}

File: AOJ/0407/meeting_place.java
import java.util.Scanner;

class Main {
  void solve(){
    Scanner sc = new Scanner(System.in);
    int N = sc.nextInt();
    int xmin = 2000, xmax = 0;
    for ( int i = 0; i < N; i++ ) {
      int x = sc.nextInt();
      if ( x < xmin ) xmin = x;
      if ( x > xmax ) xmax = x;
    }

    System.out.println( xmax - (xmin + xmax) / 2 );
  }

  public static void main(String[] args) {
    new Main().solve();
  }
}
